src/GitExtensions.GerritPlugin/FormGitReview.cs

```diff
@@ -26,6 +26,17 @@ public sealed partial class FormGitReview : GitExtensionsForm, IGitUICommandsSou
         private readonly TranslationString _saveFileQuestionCaption =
             new TranslationString("Save changes?");
 
+        private readonly TranslationString _description = new TranslationString($"Provide the configuration for{Environment.NewLine}" +
+            $"gitreview to setup Gerrit.{Environment.NewLine + Environment.NewLine}" +
+            $"Example configuration{Environment.NewLine + Environment.NewLine}" +
+            $"[gerrit]{Environment.NewLine}" +
+            $"host = review.example.com{Environment.NewLine}" +
+            $"port = 29418{Environment.NewLine}" +
+            $"project = department / project.git{Environment.NewLine}" +
+            $"defaultbranch = master{Environment.NewLine}" +
+            $"defaultremote = review{Environment.NewLine}" +
+            $"defaultrebase = 0");
+
```

Comment:
Please use verbatim string `@$"..."` instead of multiple concatenations.

Reply:
done

Reply:
Are you sure this is correct? Verbatim string preserves newline chars, so `{Environment.NewLine}` shouldn't be required.